config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/FullBundleCreator.java

```diff
@@ -120,7 +121,25 @@ public boolean test(Map<String, String> dependentBundleMap) {
                 }
             });
 
-            jsonFileUtils.createBundleMetadataFile(bundleMetadata, bundleEnvironmentValues.getLeft(), new File(bundleFolderPath));
+            //clean up intermediate file
+            cleanIntermediateFile(bundleFolderPath, bundleEnvironmentValues.getLeft() + JsonFileUtils.METADATA_FILE_NAME_SUFFIX);
+
+            String bundleMetaFileName = bundleEnvironmentValues.getLeft();
+            if (StringUtils.isNotBlank(projectInfo.getVersion())) {
+                bundleMetadata.put("version", projectInfo.getVersion() + PREFIX_FULL);
+                bundleMetaFileName = bundleMetaFileName + PREFIX_FULL;
+            }
```

Comment:
Here again, we will ignore `full` as well if `version` is not configured?

Reply:
yes